UpdateMe/app/src/main/java/nyc/c4q/syd/updateme/MainActivity.java

```diff
@@ -36,4 +57,70 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
         return super.onOptionsItemSelected(item);
     }
+    private class myAsyncTask extends AsyncTask<String, String, String>{
```

Comment:
Is it possible to put any of this in a separate class? Just because we will all have code in Main activity and it will be really long